tools/code_analyzer/op_dependency.cpp

```diff
@@ -510,16 +533,32 @@ class OpDependency : public ModulePass {
   }
 
   // Find out operator names and function pointers that are transitively
-  // connected to the same 'src' value.
+  // connected to the same 'src' instruction.
   static void scanOpSchemaStrAndFunction(
-      Value* src, const VALUE_SET& blocked,
+      Instruction* src, const VALUE_SET& blocked,
+      const std::string& contextualNamespace,
       SET* visitedOps, SET* visitedFunctions) {
     std::shared_ptr<VALUE_MAP> debugPath =
         (Verbose > 2 ? std::make_shared<VALUE_MAP>() : nullptr);
     auto callback = [&](Value* V) -> void {
-      if (auto schemaStr = extractOpSchema(V)) {
+      if (auto schemaStr = extractOpSchema(contextualNamespace, V)) {
         if (visitedOps) {
-          (*visitedOps).insert(*schemaStr);
+          // NB: Some debug string constants might be connected to the
+          // registration instruction, e.g.: "Lambda". Since we have factored
```

Comment:
So, I can add a prefix to the "debug" strings, they're totally unstructured and I can put whatever I want in them